tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx

```diff
@@ -0,0 +1,132 @@
+---
+meta:
+  title: Transforming images in an S3 bucket using Serverless Functions and Triggers - Set up
+  description: This page shows you how to set up your environment to transform images in an S3 bucket using Serverless Functions and Triggers
+content:
+  h1: Transforming images in an S3 bucket using Serverless Functions and Triggers - Set up
+  paragraph: This page shows you how to set up your environment to transform images in an S3 bucket using Serverless Functions and Triggers
+categories:
+  - serverless
+  - object storage
+tags: serverless serverless-functions functions serverless-triggers triggers sqs-queue bucket
+dates:
+  validation: 2023-07-07
+  posted: 2023-07-07
+---
+
+## Introduction
+
+This tutorial is the first of a series of two. You will learn how to set up your environment, [then how to create and deploy functions](/tutorials/create-deploy-serverless-functions/) to resize images stored in a bucket and push them to another bucket.
+
+<Lightbox src="scaleway-faas-image-transform.webp" alt="" />
+
+Serverless Functions are an asynchronous microservices architecture where event sources are decorrelated from event consumers.
+
+They work best when they are triggered by events stored in queues, which means that they can be edited without having to modify every microservices in the process.
+
+In this tutorial, you will create the following resources to use your functions:
+
+- A project dedicated to this tutorial
+- Two buckets to store media
+- A queue to manage events
+- IAM credentials to secure your environment
+- Cockpit to monitor your functions
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+  - You have around 10 JPG or PNG images.
+  - You have [Vim](https://www.vim.org/download.php) installed.
+</Message>
+
+## Creating a New Project
+
+1. Click the **Projects** tab from the [Organization Dashboard](https://console.scaleway.com/organization). A list of your Projects displays.
+2. Click **+ Create Project** on the righthand side. A pop-up appears. 
+3. Name the project "Functions Tutorial".
+4. Click **Create Project**. The new Project is added to the list. 
+
+## Creating the source bucket
+
+1. Click **Object Storage** under **Storage** on the left side menu of the console. The Object Storage dashboard displays.
+3. Click **+ Create a bucket**.
+4. Name it "source-images-<YOUR_NAME>".
+5. Chose the **AMSTERDAM** Region.
+6. Set the bucket visibility to **Private**.
+7. Click **Create Bucket**. The bucket **Files** tab displays.
+8. Upload your images to the bucket and select the **Standard** storage class.
+
+## Creating the destination bucket
+
+1. Click **Storage**, then **Object Storage** on the left side menu of the console. The Object Storage dashboard displays.
+2. Click **+ Create a bucket**.
+3. Name it "dest-images-<YOUR_NAME>".
```

Comment:
```suggestion
3. Name it `dest-images-<YOUR_NAME>`.
```
in the future, you can also just add a link to the create a bucket procedure to avoid duplicates :D